basyx/aas/adapter/http.py

```diff
@@ -218,6 +218,22 @@ def is_stripped_request(request: Request) -> bool:
 
 T = TypeVar("T")
 
+ENCODING = "utf-8"
```

Comment:
Maybe rename this to `BASE64URL_ENCODING` to make it clear, that it is only used for bas64 url encoding and decoding and not for other encoding processes, e.g. xml.